docs/sdk/1-workshop/03-attester/02-did.md

```diff
@@ -68,53 +65,22 @@ Create a file `attester/generateKeypairs.js` and copy the code below.
   {GenerateKeypairs}
 </CodeBlock>
 
-Now we have all we need to create our `DID`: a `keystore`, `keypairs` and `account`. Create
-a file `attester/createFullDid.js` and paster the following code.
+Once we have created all the necessary keys for a DID we can create the DID on chain.
+To create a DID we first initialize everything as we always do.
```

Comment:
```suggestion
To create a DID we first initialize everything.
```

Reply:
If you want to add something here, you should add a reason why we need to do this.